assimilation_code/modules/observations/default_quantities_mod.f90

```diff
@@ -0,0 +1,374 @@
+! QTY_SKIN_TEMPERATURE              units="K"         
+! QTY_SMALL_SCALE_STATE             desc="state varies with small time/space scale"
+! QTY_SMOKE
+! QTY_SNOWCOVER_FRAC                
+! QTY_SNOW_DEPTH
+! QTY_SNOW_GRAIN_SIZE
+! QTY_SNOW_MIXING_RATIO
+! QTY_SNOW_NUMBER_CONCENTR
+! QTY_SNOW_TEMPERATURE
+! QTY_SNOW_THICKNESS                
+! QTY_SNOW_WATER                    
+! QTY_SO2
+! QTY_SOIL_CARBON                   
+! QTY_SOIL_ICE
+! QTY_SOIL_LIQUID_WATER
+! QTY_SOIL_MOISTURE
+! QTY_SOIL_NITROGEN                 
+! QTY_SOIL_RESPIRATION_FLUX
+! QTY_SOIL_TEMPERATURE              units="K"     desc="is soil temp really in K or C?"
+! QTY_SOLAR_INDUCED_FLUORESCENCE
+! QTY_SOM_TEMPERATURE
+! QTY_SOOT
+! QTY_SOURCE_PHASE
+! QTY_SPECIFIC_DIFFERENTIAL_PHASE
+! QTY_SPECIFIC_HUMIDITY
+! QTY_STATE_VARIABLE                desc="basic item in a state"
+! QTY_STATE_VAR_POWER               desc="raising a state value to a power"
+! QTY_STEM_AREA_INDEX
+! QTY_STEM_CARBON                   
+! QTY_STEM_NITROGEN                 
+! QTY_STICKINESS
+! QTY_STREAM_FLOW
+! QTY_STREAM_HEIGHT
+! QTY_SULFATE
+! QTY_SULPHATED_DROPS
+! QTY_SURFACE_ALBEDO
+! QTY_SURFACE_ELEVATION             units="m"         
+! QTY_SURFACE_EMISSIVITY
+! QTY_SURFACE_HEAD
+! QTY_SURFACE_PRESSURE              units="hPa"    minval=0.0          
+! QTY_SURFACE_RUNOFF
+! QTY_SURFACE_TYPE
+! QTY_TEMPERATURE                   units="K"      minval=0.0          
+! QTY_TEMPERATURE_ELECTRON
+! QTY_TEMPERATURE_ION
+! QTY_THC
+! QTY_TOTAL_ELECTRON_COUNT
+! QTY_TOTAL_PRECIPITABLE_WATER
+! QTY_TOTAL_WATER_STORAGE           
+! QTY_TRACER_1_MIXING_RATIO
+! QTY_TRACER_2_MIXING_RATIO
+! QTY_TRACER_CONCENTRATION
+! QTY_TRACER_SOURCE
+! QTY_TURBULENT_KINETIC_ENERGY
+! QTY_UNCONFINED_WATER
+! QTY_UNDER_RUNOFF
+! QTY_U_CURRENT_COMPONENT
+! QTY_U_SEAICE_COMPONENT
+! QTY_U_WIND_COMPONENT              units="m/s"       
+! QTY_VAPOR_MIXING_RATIO
+! QTY_VEGETATED_AREA_FRACTION
+! QTY_VEGETATION_TEMPERATURE        
+! QTY_VELOCITY                      units="m/s"       minval=0.0
+! QTY_VELOCITY_U
+! QTY_VELOCITY_U_ION
+! QTY_VELOCITY_V
+! QTY_VELOCITY_VERTICAL_N2
+! QTY_VELOCITY_VERTICAL_N4S
+! QTY_VELOCITY_VERTICAL_NO
+! QTY_VELOCITY_VERTICAL_O2
+! QTY_VELOCITY_VERTICAL_O3P
+! QTY_VELOCITY_V_ION
+! QTY_VELOCITY_W
+! QTY_VELOCITY_W_ION
+! QTY_VERTICAL_TEC
+! QTY_VERTICAL_VELOCITY             units="m/s"       
+! QTY_VERTICAL_VORTICITY
+! QTY_VERTLEVEL
+! QTY_VOLCANIC_ASH
+! QTY_VORTEX_LAT                    units="degrees"    
+! QTY_VORTEX_LON                    units="degrees"    
+! QTY_VORTEX_PMIN                   units="hPa"        
+! QTY_VORTEX_WMAX                   units="m/s"        
+! QTY_V_CURRENT_COMPONENT
+! QTY_V_SEAICE_COMPONENT
+! QTY_V_WIND_COMPONENT              units="m/s"       
+! QTY_WATER_TABLE_DEPTH             
+! QTY_WATER_TYPE
+! QTY_WIND_FETCH
+! QTY_WIND_TURBINE_POWER
+! QTY_W_CURRENT_COMPONENT
+! QTY_X_LAMBDA
+!
+! END DART PREPROCESS QUANTITY DEFINITIONS
```

Comment:
TODO: check against DEFAULT kinds on the master branch. (Helen)

Reply:
The default_quantites_mod.f90 quantities does have all the quantities from the master branch DEFAULT_obs_kind_mod.F90